packages/panzer/disc-fe/src/Panzer_L2Projection.cpp

```diff
@@ -0,0 +1,8 @@
+// @HEADER
+// @HEADER
+
+#include "PanzerDiscFE_config.hpp"
+#include "Panzer_L2Projection.hpp"
+#include "Panzer_L2Projection_impl.hpp"
+
+template class panzer::L2Projection<int,panzer::Ordinal64>;
```

Comment:
Can you build this for `<int,int>` too?

Reply:
yes - panzer::Ordinal64 is a configure time chosen integer type that is set based on what Tpetra supports (if being built with ETI).